docs/DeveloperGuide.adoc

```diff
@@ -849,9 +849,16 @@ _{More to be added}_
 [appendix]
 == Non Functional Requirements
 
-.  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `1.8.0_60` or higher installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  The system should work on any <<mainstream-os, mainstream OS>> as long as it has Java `1.8.0_60` or higher installed.
```

Comment:
I left the OS part empty because I realised we have yet to discuss which OSes we are targeting. If everyone is agreeable on Windows then I'll add it in! (Not sure whether we should add macOS though.)

Reply:
I'm not sure about macOS too

Reply:
I just checked the admin section of CS2103T website. It is stated under project contraints:

"The software should work on the **Windows, Linux, and OS-X platforms**. Even if you are unable to manually test the app on all three platforms, consciously avoid using OS-dependent libraries and OS-specific features."

Reply:
Thanks @Aquarinte; I'll update it accordingly!